.changes/v2.10.0/536-improvements.md

```diff
@@ -0,0 +1,2 @@
+* `resource/vcd_vapp_vm` allows toggle memory and vCPU hot add with `cpu_hot_add_enabled` and `memory_hot_add_enabled` [GH-536]
+* `resource/vcd_vapp_vm` allows change `network` parameters without VM power off [GH-536]
```

Comment:
Looks like we're missing an entry on guest and metadata changes.
Also, same needed in docs.

Reply:
for metadata and guest properties - nothing changed. Only code refactor. Previously also was changed without VM restart. So from user perspective - no changes